src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./styles.css";
+import { Routes, Route } from "react-router-dom";
+import Navigation from "./components/Navigation";
+
+import Home from "./components/Home";
+import ProductListing from "./components/ProductListing"
+import Cart from "./components/Cart";
+import WishList from "./components/WishList";
```

Comment:
Make a separate index.js in components to export all the files from the components folder

Reply:
done